root/statistics/Editors.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {l_statistics} from '../static/scripts/common/i18n/statistics';
+import EditorLink from '../static/scripts/common/components/EditorLink';
+import {withCatalystContext} from '../context';
+import loopParity from '../utility/loopParity';
+import manifest from '../static/manifest';
+
+import {formatCount, LinkSearchableProperty} from './utilities';
+import StatisticsLayout from './StatisticsLayout';
+import type {EditorsStatsT} from './types';
```

Comment:
This type should probably be defined within this file as it is not used elsewhere.

The same holds for most other types defined in this pull request.